### DISCUSSION TITLE
Why can't I continue with the verification?

### DISCUSSION THREAD

**Discussion by MichaelYeung2004** [community/community#114547] [GitHub Education]
### Hi everyone,

<img width="272" alt="屏幕截图 2024-03-28 103016" src="https://github.com/community/community/assets/162432384/c1c7d0a7-7703-4669-abf5-9f7dd8a17491">

**Comment by HeyFang** [comment]
Did you try everyone of those?

**Comment by samus-aran** [comment]
Hi @MichaelYeung2004 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for.

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
